woocommerce/woocommerce
# Re-introduce woocommerce_product_csv_importer_check_import_file_path as deprecated

The hook was inadvertendlty removed in PR #32460.
Reintroducing it as a deprecated hook in favor of
woocommerce_csv_importer_check_import_file_path.

### All Submissions:


### Changes proposed in this Pull Request:

The `woocommerce_product_csv_importer_check_import_file_path` was inadvertently removed in #32460. Here we reintroduce it but as deprecated hook.

### How to test the changes in this Pull Request:

1. Add this snippet to the WooCommerce code, for example in `woocommerce.php`:

```php
function test_hook_deprecation(){
	add_filter('woocommerce_product_csv_importer_check_import_file_path', function ($value) {
		echo("Old filter called with: $value\n");
		return $value . '_modified';
	});

	$value = apply_filters('woocommerce_csv_importer_check_import_file_path', 'thefilename');
	echo("Result of the new filter call: $value\n");
};
```

2. Execute the code with `wp eval 'test_hook_deprecation();'`, you should get the following:

```
PHP Deprecated:  woocommerce_product_csv_importer_check_import_file_path is <strong>deprecated</strong> since version 6.5.0! Use woocommerce_csv_importer_check_import_file_path instead. in /home/konamiman/wordpress/wp-includes/functions.php on line 5691
Old filter called with: thefilename
Result of the new filter call: thefilename_modified
```

### Other information:


### FOR PR REVIEWER ONLY:
